cmd/cli/version/version.go

```diff
@@ -104,13 +119,84 @@ func (oV *VersionOptions) Run(ctx context.Context, cmd *cobra.Command) error {
 	if !oV.ClientOnly {
 		serverVersion, err := util.GetAPIClient(ctx).Version(ctx)
 		if err != nil {
-			log.Ctx(ctx).Error().Err(err).Msgf("could not get server version")
-			return err
+			return fmt.Errorf("error running version: %w", err)
 		}
 
 		versions.ServerVersion = serverVersion
 		columns = append(columns, serverVersionColumn)
+
+		clientID, err := config.GetClientID()
+		if err != nil {
+			return fmt.Errorf("error getting UserID: %w", err)
+		}
+
+		updateCheck, err := checkForUpdates(ctx, versions.ClientVersion, versions.ServerVersion, clientID)
+		if err != nil {
+			log.Ctx(ctx).Warn().Err(err).Msg("Failed to perform update check")
+		} else {
+			versions.UpdateMessage = updateCheck.Message
+			versions.LatestVersion = updateCheck.Version
+			columns = append(columns, latestVersionColumn)
+
+			// Print the update message only if --output flag is not used
+			if oV.OutputOpts.Format == output.TableFormat {
+				fmt.Println(updateCheck.Message)
```

Comment:
I believe the `cmd` var here, which is a pointer to a `cobra.Command` and so you can use https://pkg.go.dev/github.com/spf13/cobra#Command.Println which will correctly ensure it goes to the correct place.

Reply:
```suggestion
				cmd.Println(updateCheck.Message)
```